src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -2081,6 +2081,7 @@ private String init(boolean initFull) {
         }
 
         displayLockInfo(dataset);
+        displayPublishMessage();
```

Comment:
As a check, does the message stay in place when you do a change on the page that updates (via ajax) the messages panel?

Reply:
It stays up when a tabular file is uploaded and stays on after the ingest lock is removed (ingest successful)